test/resources/openssl.resource

```diff
@@ -0,0 +1,20 @@
+*** Settings ***
+Documentation       Keywords for using the openssl command line.
+...
+
+Resource            common.resource
+
+
+*** Keywords ***
+Openssl
+    [Documentation]    openssl command wrapper
+    [Arguments]    ${cmd}    ${cmd2}=
+    ${result}=    Run Process    openssl ${cmd} ${cmd2}    shell=True    # robotcode: ignore
+    Should Be Equal As Integers    ${result.rc}    0
```

Comment:
This function seems to be superficial. Do we need it?
Can we use `Command Should Work` instead?

Reply:
Command Should Work is using SSHLibrary,
we generate the certificates locally so we run shell commands directly.

Reply:
notice that this function consist of two parameter, 
since the arguments for openssl tends to be very long, this is to break the function  invocation into two lines.